src/Generators/StatementGenerator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Blueprint\Generators;
+
+use Blueprint\Contracts\Generator;
+
+abstract class StatementGenerator implements Generator
+{
+    /**
+     * @var \Illuminate\Contracts\Filesystem\Filesystem
+     */
+    protected $files;
+
+    /**
+     * @var string
+     */
+    protected $new_instance = 'new instance';
+
+    public function __construct($files)
+    {
+        $this->files = $files;
+    }
+
+    protected function buildConstructor($statement)
+    {
+        static $constructor = null;
```

Comment:
@devmsh what was the point of the `static` here? It's causing issues with tests in PHP 8.1.